internal/fsutil/create.go

```diff
@@ -121,6 +125,28 @@ func createSymlink(o *CreateOptions) error {
 	return os.Symlink(o.Link, o.Path)
 }
 
+func createHardLink(o *CreateOptions) error {
+	debugf("Creating hard link: %s => %s", o.Path, o.Link)
+	targetInfo, err := os.Lstat(o.Link)
```

Comment:
We also need a test when we hardlink to a softlink. For example, using `os.Stat` here would be an error because the file will be hardlinked to the target of the softlink and not the softlink itself.

Reply:
Test added in [ffd9951](https://github.com/letFunny/chisel/pull/17/commits/ffd99512f315c63893ae49055f3ad630d6238dd6)